### PR TITLE
add section for users to readme

### LINKED ISSUE #271: Add information for Users to the README ℹ
### Prerequisites

- [x] I have checked the existing Issues for a duplicate of this ticket
- [x] I have checked the roadmap in the documentation and am convinced this Issue fits the plans for future releases
- [x] This Issue will help us achieve at least one of the existing User Stories for the product (directly or indirectly)

### Expected Behavior

Collaborators to Kindly should have an understanding of the user's perspective of the product and may even in fact become users themselves.

The documentation should include a section about:
- what kindly is
- who it's been designed for
- how to get started _using it_

### Current Behavior

Currently we have a lot of information for potential collaborators to the project but not much for users apart from an introduction to Trafalgar Girls

### Any other notes

This should probably be a paragraph / section in the `README.md` but could take some other format if there are any interesting examples out there.

### Assignment

This issue is free for anyone to take

### PR DESCRIPTION
**Pull request by eliazzo**
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review on my code
- [x] I have made corresponding changes to the documentation (if any were needed)
- [x] My changes generate no new warnings or errors
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and previously existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

# Description

Closes #271  

### Changes to Documentation

Some of the information the issue requested we introduce was already in the readme, but I have changed the layout to make it a little clearer.

What do you think of this:
"Kindly is currently in beta stage, meaning that it is being tested by a small community of volunteer testers. If you want to be a part of the testing community, contact XYZ"

Is the testing community open to anyone who wants to get involved? If so, should I put the trafalgar girls contact details in here?

### Files changed
readme (see above)

# Tests

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
 <h1 align=center><img src=https://github.com/enBloc-org/kindly/assets/114364165/ec1855ea-13b7-4d61-800d-ff1f0c90ff70 height=200/></h1>
-Kindly is an open source web application created by Trafalgar Girls and enBloc, supporting Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
-<br>
-<br>
+
+Kindly is an open source web application created by Trafalgar Girls and <a href="https://github.com/enBloc-org">enBloc</a> that supports Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
+
+
+- ### Who is it for?
+
```
Comment (nichgalzin) [inline README.md:7]: I think sell and buy are inaccurate here.  So would recommend removing.

File: README.md [hunk 2]
```diff
@@ -1,13 +1,29 @@
...
+- ### Who are we?
+
+   **Trafalgar Girls** is a volunteer project providing informational, practical and emotional support to Ukrainian Refugees living in Great Britain and beyond.
+   
+   **enBloc** is a web agency born from the ever growing Founders and Coders alumni community and a creator of open source tools. Our ethos is simple: inspire, collaborate, and grow together.
+
+
+- ### How to start using it?
+
+   Kindly is currently in beta stage, meaning that it is being tested by a small community of volunteer testers. If you want to be a part of the testing community, contact **XYZ**
```
Comment (nichgalzin) [inline README.md:19]: Should replace XYZ with maybe a link to the discord server? Or a trafalgar girls email? Not sure

**Review by nichgalzin — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eliazzo) [reply README.md:7]: Kindly is for anyone who wants to share items with the Ukrainian refugee community. Equally, it is for members of the Ukrainian refugee community who want to borrow or receive items

File: README.md [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
-Kindly is an open source web application created by Trafalgar Girls and enBloc, supporting Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
-<br>
-<br>
+
+Kindly is an open source web application created by Trafalgar Girls and <a href="https://github.com/enBloc-org">enBloc</a> that supports Ukrainian refugees in the United Kingdom. Individuals can contribute items at no cost to provide much-needed support to members of the Ukrainian refugee community.
+
+
+- ### Who is it for?
+
+   Kindly is for anyone who wants to share items with the Ukrainian refugee community. Equally, it is for members of the Ukrainian refugee community who want to borrow or receive items
```
Comment (eliazzo) [inline README.md:8]: to give back, borrow or receive items

File: README.md [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
-We **welcome** your contributions. Please familiarize yourself with our documentation before getting started. You can find guidance on how to contribute in the following sources:
+Kindly is an open source project and we welcome anyone looking to get involved. Please familiarise yourself with our documentation before getting started. You can find guidance on how to contribute in the following sources:
 
 - Our [discord server](https://discord.gg/Mrh5NYw52S) - say Hi to the other developers and share knowledge here 👋
 - [Before your first issue](https://github.com/enBloc-org/kindly/blob/dev/.github/BEFORE_YOUR_FIRST_ISSUE.md)
 - [Contributing](https://github.com/enBloc-org/kindly/blob/dev/.github/CONTRIBUTING.md)
 - [Best practice](https://github.com/enBloc-org/kindly/blob/dev/.github/BEST_PRACTICE.md)
 - [Testing](https://github.com/enBloc-org/kindly/blob/dev/.github/TESTING.md)
 - [Privacy](https://github.com/enBloc-org/kindly/blob/dev/PRIVACY.md)
+- [Governance](https://github.com/enBloc-org/kindly/blob/dev/GOVERNANCE.md)
```
Comment (eliazzo) [inline README.md:34]: For details on the different roles you can take on

**Review by nichgalzin — APPROVED** [review]
Looking good!  🧞‍♂️

**Pull request merged**